### ISSUE TITLE
Create `make_table_32_gtsum`

### ISSUE THREAD

**Issue by aassuied-ps** [pharmaverse/falcon#113]
Create function `make_table_32_gtsum` to build table 32 using the `gt` and `gtsummary` packages.
See #17 and https://pharmaverse.github.io/falcon/quarto/table-templates/template-table_32.html for specifications for this table.

The table output should be very similar to the output created by `make_table_32`.

The Table 32 template page on the falcon site will be updated once the function is ready.

**Comment by aassuied-ps** [comment]
A first draft has been posted. I saw there is an issue on the `add_overall()` function. We can not enter the label we want so far. See here: https://github.com/ddsjoberg/gtsummary/issues/1505

Should we modify that and add the label of the overall column in the `modify_header()` function? Other `gtsummary` tables we have are currently using `add_overall()`.

Is there a way to use our `alt_counts_df` that we're using with `rtables` and `tern`? For now what I did is joining ADVS with the population table (ADSL).
